analyzers/src/SonarAnalyzer.CSharp/Rules/Hotspots/ExecutingSqlQueries.cs

```diff
@@ -74,12 +66,11 @@ public ExecutingSqlQueries()
                 : null;
 
         protected override bool IsConcat(ExpressionSyntax argument, SemanticModel semanticModel) =>
-            IsStringMethodInvocation("Concat", argument, semanticModel) ||
-            (
-                argument.IsKind(SyntaxKind.AddExpression) &&
-                argument is BinaryExpressionSyntax concatenation &&
-                !IsConcatenationOfConstants(concatenation, semanticModel)
-            );
+            IsStringMethodInvocation("Concat", argument, semanticModel)
+            || IsConcatenation(argument, semanticModel)
+            || (argument is IdentifierNameSyntax identifierNameSyntax
```

Comment:
While you've extracted 3 `&&` conditions to `IsConcatenation`, this new 3 `&&` condition would also make sense to be extracted to a method to keep the code structure regular.